keras/losses.py

```diff
@@ -1761,6 +1761,24 @@ def _ragged_tensor_sparse_categorical_crossentropy(y_true,
       sparse_categorical_crossentropy, from_logits=from_logits, axis=axis)
   return _ragged_tensor_apply_loss(fn, y_true, y_pred, y_pred_extra_dim=True)
 
+def _check_ground_truth_label_range(y_true):
+  """Checks whether ground truth label lies inside of the range i.e.,
+  either 0 or 1.
+  
+  Args:
+    y_true: Ground truth values. shape = `[batch_size, d0, .. dN]`.
+
+  Returns:
+    Boolean value `True` if ground truth labels lie inside of the range,
+    `False` otherwise.
+  """
+  flatten_y_true_unpacked = tf.unstack(tf.reshape(y_true, [-1]))
+  for label in flatten_y_true_unpacked:
```

Comment:
The for loop here will have very bad performance, and usually for tf in general, we would like to leverage tf ops to do bulk computations. Could u try compare the whole flatten_y_true_unpacked with zero and one, and count how many of them matches?